setup.cfg

```diff
@@ -90,8 +90,3 @@ convention = pep257
 # D202: Ignore a blank line after docstring (collision with Python Black in decorators)
 add-ignore = D104,D107,D202
 max-line-length = 120
-
-[tool:isort]
```

Comment:
why canot use this config? :]

Reply:
Using the `pyproject.toml` config (or `.isort.cf`) is considered the [preferred format by isort](https://pycqa.github.io/isort/docs/configuration/config_files/). `setup.cfg` is considered more of a legacy format, so I thought it would make sense to use the more future-proof format. Also, I searched through the code base and couldn't find any existing reference to isort, so I assumed this wouldn't be considered a breaking change.